test/remote/certificate_sign_tests.js

```diff
@@ -42,6 +42,7 @@ TestServer.start(config)
             t.ok(payload['fxa-generation'] > 0, 'cert has non-zero generation number')
             t.ok(new Date() - new Date(payload['fxa-lastAuthAt'] * 1000) < 1000 * 60 * 60, 'lastAuthAt is plausible')
             t.equal(payload['fxa-verifiedEmail'], email, 'verifiedEmail is correct')
+            t.equal(payload['fxa-tokenVerified'], true, 'tokenVerified is correct')
```

Comment:
It's critical that non-verified sessions produce `fxa-tokenVerified === false` in the certificate, so I think it would be worth adding an explicit test case that to exercise it, i.e. one that signs a certificate using an unverified session token.